### PR TITLE
Exec September 24 2021

### PR DESCRIPTION
**Pull request by prose11**
Spell Copy for tomorrow's Executive

### CODE REVIEW

File: governance/votes/Executive vote - September 24, 2021.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- Multiple **test actions** will take place for Layer 2 DAI solutions on Optimism and Arbitrum, with the intention of denying the Protocol Engineering Core Unit's Deployer Keys if successful. 
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - September 24, 2021.md:32]: We have office hours for this? If so, we should maybe remove the reason here, as there are no new collateral types.

File: governance/votes/Executive vote - September 24, 2021.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  - ETH-A Local Liquidation Limit from 30 million DAI to **40 million DAI**
+  - ETH-B Local Liquidation Limit from 15 million DAI to **25 million DAI**
+  - ETH-C Local Liquidation Limit from 20 million DAI to **30 million DAI**
+  - WBTC-A Local Liquidation Limit from 15 million DAI to **25 million DAI**
+
+For more information on the rationale behind these proposed changes, please view [this post](https://forum.makerdao.com/t/adjusting-auction-parameters-for-eth-a-eth-c-and-wbtc-a-vault-types/10331) from the Risk Core Unit. 
+
+### Layer 2 Test Actions
+
+If this executive proposal passes, it will test the configuration of L2 components, including the governance relay and L2 DAI. Once it is confirmed that Governance has control over all L2 systems, the Protocol Engineering Core Unit will **manually** `deny` its deployer addresses, meaning Governance will be the only authority retaining full control of the token bridges.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - September 24, 2021.md:55]: Should we have a link to a PE forum post for this?

Comment (LongForWisdom) [reply governance/votes/Executive vote - September 24, 2021.md:55]: Like even I'm not 100% sure what we're doing from this explanation. I think:
* Ensuring that Maker Governance is able to make changes to DAI bridges on both sides.

What are we testing about L2 DAI? I wasn't aware we could do anything to it?

**Review by LongForWisdom — CHANGES_REQUESTED** [review]

**Comment by LongForWisdom** [comment]
Good to merge once comments are addressed.

File: governance/votes/Executive vote - September 24, 2021.md (hunk 1 above)
Comment (prose11) [reply governance/votes/Executive vote - September 24, 2021.md]: ✅

File: governance/votes/Executive vote - September 24, 2021.md (hunk 2 above)
Comment (prose11) [reply governance/votes/Executive vote - September 24, 2021.md]: I think this covers it, post is up and now linked

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**